app/models/post.rb

```diff
@@ -14,6 +14,6 @@ def update_posts_counter
   end
 
   def recent_comments
-    comments.order(created_at: :desc).limit(5)
+    comments.includes(:author).order(created_at: :desc).limit(5)
```

Comment:
* Kindly solve the N + 1 problem  when fetching all posts and their comments for a user, you can do this in your posts controller index action [Check requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Make%20sure%20that%20the%20N%2B1%20problem%20is%20solved%20when%20fetching%20all%20posts%20and%20their%20comments%20for%20a%20user.%20This%20is%20the%20corresponding%20wireframe)

Reply:
This change has been done well